tests/test_route53resolver/test_route53resolver_endpoint.py

```diff
@@ -143,6 +146,7 @@ def test_route53resolver_invalid_create_endpoint_args():
     ) in err["Message"]
 
     subnet_too_long = [{"SubnetId": "a" * 33, "Ip": "1.2.3.4"}]
+    print(subnet_too_long)
```

Comment:
Should the print() be here?  

Thanks for adding the necessary changes to support the newer botocore!

Reply:
Good catch @kbalk! I'll push a fix.